gradle/libs.versions.toml

```diff
@@ -2,23 +2,23 @@
 
 [versions]
 accompanist = "0.28.0"
-agp = "8.1.2"
+agp = "8.2.1"
 androidCompileSdk = "34"
 androidMinSdk = "21"
 androidTargetSdk = "32"
 androidx-lifecycle = "2.6.1"
 androidx-navigation-compose = "2.5.1"
 coil = "2.2.1"
-composePlugin = "1.5.3"
+composePlugin = "1.5.11"
 composeBom = "2023.10.01"
-composeCompiler = "1.5.3"
+composeCompiler = "1.5.7"
 coroutines = "1.6.4"
-dependencyAnalysis = "1.13.1"
+dependencyAnalysis = "1.29.0"
 detekt = "1.21.0"
 junit5 = "5.8.2"
 jvmTarget = "11"
-kotlin = "1.9.10"
-ksp = "1.9.10-1.0.13"
+kotlin = "1.9.21"
+ksp = "1.9.21-1.0.16"
```

Comment:
Kotlin `1.9.20` is skipped due to an [issue](https://github.com/google/ksp/issues/1591) with KSP that makes it impossible to compile the library with this version.